tests/formspree_test_case.py

```diff
@@ -0,0 +1,34 @@
+
+import os
+import fakeredis
+
+from flask.ext.testing import TestCase
+import formspree
+from formspree import create_app
+from formspree import settings
+from formspree.app import DB, REDIS
+
+
+class FormspreeTestCase(TestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        ''' configures app for testing '''
+        formspree.app.get_redis = lambda : fakeredis.FakeStrictRedis()
```

Comment:
These are good points. If you can submit a patch using mock instead I'll accept it. This was a first iteration away from the previous method which had a lot more test specific code. I checked out mock but stopped shy of implementing it.